src/components/organisms/Onboarding/StepBody.module.css

```diff
@@ -0,0 +1,51 @@
+.content {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  width: 100%;
+}
+
+.cardContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.cardContainer,
+.image {
+  width: 100%;
+}
+
+.image {
+  object-fit: contain;
+  max-height: 25rem;
+}
+
+.actions {
+  display: flex;
+  flex-direction: column;
+  gap: calc(var(--spacer) / 2);
+  width: 100%;
+}
+
+.card {
+  composes: box from '../../atoms/Box.module.css';
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 20rem;
+  min-height: 20rem;
+  height: 100%;
+}
+
+@media screen and (min-width: 60rem) {
```

Comment:
question(blocking): Does the order follow any logic? If not, I'd just go for an alphabetic order here.

Reply:
They are supposed to be ordered by the tag depth, but there is a mistake since `card` should come before `actions`. The order should be `content`, then `image` and `cardContainer` at the same level, then `card` and at the end `actions`.
Maybe it does not make much sense though and we should still switch to the alphabetical order

Reply:
I am fine with both, just make sure that it follows a given logic. 
Hint: Alphabetical order is easier to understand for someone who wants to enhance this file and does not know anything about it's usage yet.

Reply:
We are not enforcing any convention as far as I know, but looking at the preexisting code it looks like there is an attempt to follow the tag depth convention, that's the reason I went with it. I agree that it's difficult to maintain over time and in a new repo I would probably go alphabetically, but I'm not sure if we should use it here since it would not be consistent with the rest of the codebase

Reply:
Agreed. And there are reasons for both attempts. Just fix the mistake and we are good to go here.